web/Employee/views.py

```diff
@@ -21,18 +21,22 @@ def staff(request, tab):
     today_reservations = Reservation.objects.filter(start_date=today)
     return_reservations = Reservation.objects.filter(end_date=today)
     car_inventory = Car.objects.all()
+    all_reservations = Reservation.objects.filter(start_date__gte=today)
     context = {
         "formatted_date": formatted_date,
         "car_inventory": car_inventory,
         "today_reservations": today_reservations,
         "return_reservations": return_reservations,
-    }
```

Comment:
This closing bracket is probably needed, I might just not be seeing it further down